Content.Shared/Body/Part/BodyPartComponent.cs

```diff
@@ -68,16 +70,44 @@ public sealed partial class BodyPartComponent : Component, ISurgeryToolComponent
     /// <summary>
     /// How much health the body part has until it pops out.
     /// </summary>
+    [ViewVariables]
+    public float Integrity => Damage.GetTotal().Float();
+
+    /// <summary>
+    /// The DamageSpecifier that contains all types of damage that the BodyPart can take.
+    /// TODO: Rework this with DamageableComponent
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public DamageSpecifier Damage = new()
+    {
+        DamageDict = new Dictionary<string, FixedPoint2>
+        {
+            { "Blunt", 0 },
+            { "Slash", 0 },
+            { "Piercing", 0 },
+            { "Heat", 0 },
+            { "Cold", 0 },
+            { "Shock", 0 },
+            { "Caustic", 0 },
+        }
+    };
+
+    [DataField, AutoNetworkedField]
+    public float MaxIntegrity = 0;
```

Comment:
Likewise to the other comment on Integrity now being a confusing label, this makes it even more confusing since we flipped the logic, its not a MaxIntegrity but a MinIntegrity, or MinDamage, whatever.